src/operations/replace-followed-threads-link.ts

```diff
@@ -1,9 +1,11 @@
 import * as SITE from "globals-site";
 import * as T from "text";
 
-export default (e: { followedThreadsLink: HTMLElement }) => {
-    const link = e.followedThreadsLink as HTMLAnchorElement;
-    link.textContent = T.general.my_posts;
+export default (e: { followedThreadsLinkText: HTMLElement }) => {
+    const text = e.followedThreadsLinkText as HTMLSpanElement;
+    const link = text.parentElement as HTMLAnchorElement;
+    const li = link.parentElement as HTMLLIElement;
+    text.textContent = T.general.my_posts;
     link.href = SITE.PATH.MY_POSTS;
-    (link.parentElement as HTMLElement).classList.remove(SITE.CLASS.hasUnread);
+    li.classList.remove(SITE.CLASS.hasUnread);
```

Comment:
I noticed that we don't need to do this if we adjust the CSS appropriately (see main review comment).

```suggestion
```